graphql/schemas/Social/userList.graphql

```diff
@@ -0,0 +1,78 @@
+input UserListInput {
+    name: String!
+    description: String
+    is_public: Boolean
+    is_default: Boolean
+}
+
+type UserList @model(class: "\\Kanvas\\Social\\UsersLists\\Models\\UserList"){
+    id: Int!
+    slug: String!
+    name: String!
+    description: String
+    is_public: Boolean
+    is_default: Boolean!
+    user: User!
+    apps: [App]
+    company: Company!
+    items: [Message]
+}
+
+extend type Mutation @guard {
+    createUserList(input: UserListInput!): UserList 
```

Comment:
@FredPeal if the below method is call addToList should this method be call createList ?